### DISCUSSION TITLE
Windows installation issue

### DISCUSSION THREAD

**Discussion by enzodepaola** [NVlabs/curobo#111] [Software Q&A] [answered]
**Hi all,
i'm trying to install curobo in an windows setup with the following properties:

NVIDIA GPU : Quadro RTX 4000
ISAAC SIM : 2022.2.1 perfectly working
CUDA_Toolkit: 11.7
Python : 3.7.13 (from isaac sim installation)

actually i'm not able to finishing compiling the installation, reading the isaac sim installation guide, due to the following errors**:

      RuntimeError: Error compiling objects for extension
            C:\Users\Siemens\Documents\Omniverse\pkg\isaac_sim-2022.2.1\kit\python\lib\site-packages\setuptools\_distutils\dist.py:988: _DebuggingTips: Problem in editable installation.
      
      C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.38.33130/include\type_traits(1965): error: class template "std::_Function_args" has already been defined

      C:/Program Files (x86)/Windows Kits/10/include/10.0.22621.0/ucrt\malloc.h(55): error: more than one instance of overloaded function "_alloca" has "C" linkage
      C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.7\include\sm_20_intrinsics.hpp(136): error: more than one instance of overloaded function "__nv_cvta_global_to_generic_impl" has "C" linkage

      C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.7\include\sm_20_intrinsics.hpp(137): warning #1581-D: declaration overloads built-in function "__nv_cvta_shared_to_generic_impl"

      C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.7\include\sm_20_intrinsics.hpp(194): error: asm operand type size(8) does not match type/size implied by constraint 'r'

**I just cutted out couples of the errors listed in the output. Probably i have something strange in my environment, but i would like to ask you if someone can help me in solving it, thank you in advance **

**Comment by balakumar-s** [comment, marked as answer]
Windows is not supported. Some users were able to install on WSL: https://github.com/NVlabs/curobo/discussions/37

**Reply by enzodepaola** [reply]
Thank you for the answer!
Then i will move forward with docker.
